src/main/kotlin/com/re/back/dtos/response/AuthResponseDto.kt

```diff
@@ -0,0 +1,14 @@
+package com.re.back.dtos.response
+
+import com.re.back.enums.UserRole
+import java.time.LocalDateTime
+
+data class AuthResponseDto(
+    val id : Int?,
+    val userName : String,
+    val email : String,
+    val role : UserRole,
```

Comment:
Let's avoid roles for now, I guess now we only have user , and we can think about the roles in the future PR